packages/vscode-extension/src/utils/try-to-catch.ts

```diff
@@ -0,0 +1,17 @@
+// COPIED FROM https://github.com/coderaiser/try-to-catch/blob/26a0ba95c71b7bdb710bb678732531dd31549c40/lib/try-to-catch.js due to import errors
+// See https://stackoverflow.com/q/68953594/13994294
+
+export default async function tryToCatch(fn: (...args: unknown[]) => any, ...args: any[]): Promise<[error: Error | null, result?: any]> {
+    check(fn)
+
+    try {
+        return [null, await fn(...args)]
+    } catch(e) {
+        return [e]
+    }
+}
+
+function check(fn: (...args: unknown[]) => any) {
+    if (typeof fn !== 'function')
+        throw Error('fn should be a function!')
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/LinqLover/downstream-repository-mining/pullRequest?prid=8058375)
```suggestion
        {throw Error('fn should be a function!')}
```